src/pages/workspaces/workspace/analysis/modals/compute-modal-components.ts

```diff
@@ -0,0 +1,71 @@
+import { div, h, label } from 'react-hyperscript-helpers'
```

Comment:
This looks like a small testable file! As we add TS files we are trying to add tests.

Some that come to mind:
For PersistentDiskType:
- Passing `diskExists` `[true,false]` to `PersistentDiskType`
- Ensuring `updateComputeConfig` gets called with proper value on change

For PersistentDiskSection:
- Passing `diskExists` `[true,false]`
- Ensuring `updateComputeConfig` gets called with proper value on change
- Ensuring `handleLearnMoreAboutPersistentDisk` gets called on click